task_hive/.github/workflows/sonarcloud.yml

```
@@ -1,40 +1,48 @@
+          SONAR_TOKEN: ${{ secrets.SONARQUBE_TOKEN }}
+        with:
+          args: >-
+            -Dsonar.host.url=https://sonarcloud.io
+            -Dsonar.projectKey=Learnathon-By-Geeky-Solutions_nstu-codebreakers
+            -Dsonar.organization=learnathon-by-geeky-solutions
+            -Dsonar.java.version=17
+            -Dsonar.verbose=true
+            -Dsonar.sources=lib
+            -Dsonar.tests=test
+            -Dsonar.dart.lcov.reportPaths=coverage/lcov.info
+            -Dsonar.sourceEncoding=UTF-8
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)